assets/base.css

```diff
@@ -1428,6 +1428,10 @@ button.shopify-payment-button__button--unbranded {
   display: flex;
```

Comment:
Yes, it will match the template that is assigned to the product. Description is never shown inside the quick modal, but it does reflect other settings.

Here's an example where each template has a different [variant picker type](https://screenshot.click/17-06-2pegd-vl2ox.mp4).